codegen/builtin_fs.go

```diff
@@ -252,10 +253,10 @@ func (l Local) Call(ctx context.Context, cln *client.Client, val Value, opts Opt
 	fs.SessionOpts = append(fs.SessionOpts, llbutil.WithSyncedDir(id, filesync.SyncedDir{
 		Name: localPath,
 		Dir:  localDir,
-		Map: func(_ string, st *fstypes.Stat) bool {
+		Map: func(_ string, st *fstypes.Stat) fsutil.MapResult {
 			st.Uid = 0
 			st.Gid = 0
-			return true
+			return fsutil.MapResultKeep
```

Comment:
What does this do?

Reply:
it is the same result, the fsutil walk api changed to require an enum to be resturns vs the `true` (to mean "keep going")